developer/system/template-manager.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: doc
+title: Template Manager
+permalink: /doc/template-manager/
+---
+
+Template Manager Design
```

Comment:
I think this h1 body title should match the filename and doc index link. I.e., either remove `Design` from this or add it to the others.

Reply:
Done (removed `Design` from the title).